core/src/test/java/org/testcontainers/images/RemoteDockerImageTest.java

```diff
@@ -28,25 +26,25 @@ public void toStringWithExceptionContainsOnlyImageNameFuture() {
         imageNameFuture.completeExceptionally(new RuntimeException("arbitrary"));
 
         RemoteDockerImage remoteDockerImage = new RemoteDockerImage(imageNameFuture);
-        assertThat(remoteDockerImage.toString(), containsString("imageName=java.lang.RuntimeException: arbitrary"));
+        assertThat(remoteDockerImage.toString()).contains("imageName=java.lang.RuntimeException: arbitrary");
     }
 
     @Test(timeout = 5000L)
     public void toStringDoesntResolveImageNameFuture() {
         CompletableFuture<String> imageNameFuture = new CompletableFuture<>();
 
         // verify that we've set up the test properly
-        assertFalse(imageNameFuture.isDone());
+        assertThat(imageNameFuture.isDone()).isFalse();
```

Comment:
nit: we could use AssertJ's support for `CompletableFuture` here and in other places (vs using `isDone()` et al)